app/analyse/util/data/general_stopwords.txt

```diff
@@ -0,0 +1 @@
+this
```

Comment:
Do you want add more word in this pr?
If not, delete it and add into next pr, or add more words in this pr.